bot.py

```diff
@@ -82,7 +82,7 @@ def __init__(self,
         self.exchange_frame = ttk.Frame(self, style="app.TFrame")
         self.exchange_menu, self.exchange_index = database_utils.get_all_exchanges()
         self.exchange_name = MyWidget(self.app, self.exchange_frame, handle="exchange", choices= self.exchange_menu)
-        self.coin = MyWidget(self.app, self.exchange_frame, handle="coin", choices= ["XMR", "LTC", "PEPE"])
+        self.coin = MyWidget(self.app, self.exchange_frame, handle="coin", choices= ["XMR", "LTC", "ETH", "DOGE","" "PEPE"])
```

Comment:
this has a typo, no? 

,"" "PEPE"])

Reply:
ya typo, weird. just wait for next commit